boxsdk/session/session.py

```diff
@@ -429,11 +435,15 @@ def _get_retry_request_callable(self, network_response, attempt_number, request)
             `callable`
         """
         # pylint:disable=unused-argument
+        data = kwargs.get('data', {})
+        grant_type = None
+        if 'grant_type' in data:
```

Comment:
I get the following error:
TypeError: a bytes-like object is required, not 'str'

The problem seems to be that `'grant_type'` is a `str`, but data is a bytes-object. So this can't work.
Can you confirm?